examples/gno.land/r/demo/boards2/public.gno

```diff
@@ -112,97 +109,75 @@ func CreateReply(bid BoardID, threadID, replyID PostID, body string) PostID {
 
 func FlagReply(bid BoardID, threadID, replyID PostID, reason string) {
 	caller := std.GetOrigCaller()
-
 	board := mustGetBoard(bid)
 	assertHasBoardPermission(board, caller, PermissionThreadFlag)
 
 	thread := mustGetThread(board, threadID)
 	reply := mustGetReply(thread, replyID)
 
-	if hide := flagItem(reply, NewFlag(caller, reason)); hide {
+	hide := flagItem(reply, NewFlag(caller, reason))
+	if hide {
```

Comment:
This was split into two lines because it seems that AVL parsing in filetests is not yet able to handle the inline syntax.

Inline syntax fails with:
`interface conversion: gnolang.Stmt is *gnolang.AssignStmt, not *gnolang.bodyStmt (code=2)`

cc @thehowl

Reply:
@jeronimoalbi can you please create an issue in Gno repo about this?

Reply:
I will 👍